src/Moryx.Products.Management/Implementation/Storage/ProductStorage.cs

```diff
@@ -473,20 +472,21 @@ public long SaveType(IProductType modifiedInstance)
         {
             using (var uow = Factory.Create())
             {
-                var entity = SaveProduct(uow, modifiedInstance);
+                var productSaverContext = new ProductPartsSaverContext(uow);
+                var entity = SaveProduct(ref productSaverContext, modifiedInstance);
```

Comment:
Since `ProductPartsSaverContext` is a class, it is passed by reference anyway. Adding `ref` would only be necessary if the reference itself, not instance properties, are modified in the method.